entrypoint.sh

```diff
@@ -7,227 +7,247 @@ set -o pipefail
 
 # shellcheck disable=SC2034
 {
-  declare -r ERROR='\033[0;31m'
-  declare -r WARNING='\033[0;33m'
-  declare -r INFO='\033[0m'
-  declare -r DEFAULT='\033[0m'
+	declare -r ERROR='\033[0;31m'
+	declare -r WARNING='\033[0;33m'
+	declare -r INFO='\033[0m'
+	declare -r DEFAULT='\033[0m'
 }
 
 function feedback() {
-  color="${1:-DEFAULT}"
-  case "${1}" in
-    ERROR)
-      >&2 echo -e "${!color}${1}:  ${2}${DEFAULT}"
-      ;;
-    WARNING)
-      >&2 echo -e "${!color}${1}:  ${2}${DEFAULT}"
-      ;;
-    *)
-      echo -e "${!color}${1}:  ${2}${DEFAULT}"
-      ;;
-  esac
+	color="${1:-DEFAULT}"
+	case "${1}" in
+	ERROR)
+		echo >&2 -e "${!color}${1}:  ${2}${DEFAULT}"
+		;;
+	WARNING)
+		echo >&2 -e "${!color}${1}:  ${2}${DEFAULT}"
+		;;
+	*)
+		echo -e "${!color}${1}:  ${2}${DEFAULT}"
+		;;
+	esac
 }
 
 function setup_environment() {
-  # Set the preferred shell behavior
-  shopt -s globstar
-
-  # Set the default branch
-  export DEFAULT_BRANCH="main"
-
-  # Set workspace to /goat/ for local runs
-  export DEFAULT_WORKSPACE="/goat"
-
-  # Create variables for the various dictionary file paths
-  export GLOBAL_DICTIONARY="/etc/opt/goat/seiso_global_dictionary.txt"
-  export REPO_DICTIONARY="${GITHUB_WORKSPACE:-/goat}/.github/etc/dictionary.txt"
-
-  # Map certain environment variables
-  if [[ "${INPUT_DISABLE_TERRASCAN:-}" == "true" ]]; then
-    export VALIDATE_TERRAFORM_TERRASCAN="false"
-  fi
-
-  if [[ "${INPUT_DISABLE_MYPY:-}" == "true" ]]; then
-    export VALIDATE_PYTHON_MYPY="false"
-  fi
-
-  if [[ -n ${INPUT_EXCLUDE:+x} ]]; then
-    export FILTER_REGEX_EXCLUDE="${INPUT_EXCLUDE}"
-  fi
-
-  if [[ "${INPUT_LOG_LEVEL:-}" =~ ^(ERROR|WARN|NOTICE|VERBOSE|DEBUG|TRACE)$ ]]; then
-    export LOG_LEVEL="${INPUT_LOG_LEVEL}"
-    export ACTIONS_RUNNER_DEBUG="true"
-  else
-    echo "The provided LOG_LEVEL of ${INPUT_LOG_LEVEL:-null or unset} is not valid"
-  fi
-
-  linter_failed="false"
-  declare -a linter_failures
-  declare -a linter_successes
+	# Set the preferred shell behavior
+	shopt -s globstar
+
+	# Set the default branch
+	export DEFAULT_BRANCH="main"
+
+	# Set workspace to /goat/ for local runs
+	export DEFAULT_WORKSPACE="/goat"
+
+	# Create variables for the various dictionary file paths
+	export GLOBAL_DICTIONARY="/etc/opt/goat/seiso_global_dictionary.txt"
+	export REPO_DICTIONARY="${GITHUB_WORKSPACE:-/goat}/.github/etc/dictionary.txt"
+
+	# Map certain environment variables
+	if [[ ${INPUT_DISABLE_TERRASCAN-} == "true" ]]; then
+		export VALIDATE_TERRAFORM_TERRASCAN="false"
+	fi
+
+	if [[ ${INPUT_DISABLE_MYPY-} == "true" ]]; then
```

Comment:
Does this actually disable mypy?  I don't think it does, does it?  It seems like we're missing a disable linter feature of `linters.json` and `entrypoint.sh`; if the provided env var depending on the linter, something like `INPUT_DISABLE_${linter[name]}`, has a value of `true` then log (or queue) a message that it was disabled and return with exit 0.

Reply:
I confirmed; it doesn't

Reply:
Added logic for disabling mypy